### PR TITLE
add a python example for cdk for terraform

### PR DESCRIPTION
**Pull request by HarshCasper**
In reference to a user query over the [Community Slack](https://localstack-community.slack.com/archives/CMAFN2KSP/p1681467073099039)

### CODE REVIEW

**Review by baermat — CHANGES_REQUESTED** [review]
Nice addition! :tada:
I only have one request, if we could add a Makefile as described in the top level README, we could include it in the pipeline. Even though we do not really check here automatically if the stack was deployed properly, it would still serve as smoke test if something major breaks :)

File: cdk-for-terraform/python/Pipfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[[source]]
```
Comment (whummer) [inline cdk-for-terraform/python/Pipfile:1]: We usually use `setup.py`/`setup.cfg` in combination with `virtualenv` to install the dependencies locally.

In this repo we're using Pipfile/pipenv, which is an alternative way to install dependencies - which is also perfectly fine, but breaks a bit with our usual standard pattern.

Wondering if we should unify and also adopt our standard `virtualenv` based setup here?

**Review by whummer — APPROVED** [review]
Great, thanks for adding this @HarshCasper ! Only one comment related to `pipenv` vs `virtualenv` - not really critical, but perhaps we could still address this before merging.. Let me know if you need any help with it.. 👍

File: cdk-for-terraform/python/Pipfile (hunk 1 above)
Comment (HarshCasper) [reply cdk-for-terraform/python/Pipfile:1]: I will do that in a follow-up PR 👍

**Comment by HarshCasper** [comment]
> I only have one request, if we could add a Makefile as described in the top level README, we could include it in the pipeline.

I will implement that once I migrate from `pipenv` to `virtualenv` as mentioned in the PR review! 😄

**Pull request merged**